File: Python/1446.py
class Solution(object):
    def maxPower(self, s):
        max_ = 0
        k = 0
        for i in range(1,len(s)):
            if s[i] != s[i-1]:
                max_ = max(max_,i-k)
                k = i
        return max(max_,len(s)-k)

File: Java/1446.java
class Solution {
    public int maxPower(String s) {
        int max = 0, k = 0,i = 0;
        for(i = 1; i < s.length(); i++){
            if(s.charAt(i) != s.charAt(i-1)){
                if(i-k > max) max = i-k;
                k = i;
            }
        }
        if(i-k > max) max = i-k;
        return max;
    }
}